app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from typing import Dict, Any
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: Dict[str, Any]) -> str | Exception:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("You are not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Expired vaccination date")
+        if not visitor.get("wearing_a_mask"):

```

Comment:
You are comparing `visitor.get("wearing_a_mask")` to `None` indirectly. According to the checklist, it is better to use `if visitor.get("wearing_a_mask") is not None:`.